examples/hysteresis.ipynb

```diff
@@ -0,0 +1,408 @@
+    "            \"K2\": me.Ku(0.0, unit=u.J / u.m**3),\n",
+    "            \"Ms\": me.Ms(1189556.9164459957, unit=u.A / u.m),\n",
+    "            \"A\": me.A(6.015043651663635e-12, unit=u.J / u.m),\n",
+    "        },\n",
+    "        {  # sphere\n",
+    "            \"theta\": 0.0,\n",
+    "            \"phi\": 0.0,\n",
+    "            \"K1\": me.Ku(0.0, unit=u.J / u.m**3),\n",
+    "            \"K2\": me.Ku(0.0, unit=u.J / u.m**3),\n",
+    "            \"Ms\": me.Ms(0.0, unit=u.A / u.m),\n",
+    "            \"A\": me.A(0.0, unit=u.J / u.m),\n",
+    "        },\n",
+    "        {  # shell\n",
+    "            \"theta\": 0.0,\n",
+    "            \"phi\": 0.0,\n",
+    "            \"K1\": me.Ku(0.0, unit=u.J / u.m**3),\n",
+    "            \"K2\": me.Ku(0.0, unit=u.J / u.m**3),\n",
+    "            \"Ms\": me.Ms(0.0, unit=u.A / u.m),\n",
+    "            \"A\": me.A(0.0, unit=u.J / u.m),\n",
+    "        },\n",
+    "    ],\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1790b30e-1e47-4de4-98ca-4c5e2d3b5f55",
+   "metadata": {},
+   "source": [
+    "### Simulation parameters\n",
+    "\n",
+    "We finally define all the simulation parameters we will use. For an exhaustive explanation of these parameters, check [the documentation](https://mammos-project.github.io/mammos/api/mammos_mumag.parameters.html#module-mammos_mumag.parameters)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "05d7139b-0331-4fe7-8810-ee844d0a615a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "par = Parameters(\n",
+    "    size=1.0e-9,\n",
+    "    scale=0,\n",
+    "    m_vect=[0, 0, 1],\n",
+    "    hstart=10,\n",
+    "    hfinal=-10,\n",
+    "    hstep=-2,\n",
+    "    h_vect=[0.01745, 0, 0.99984],\n",
+    "    mstep=0.4,\n",
+    "    mfinal=-1.2,\n",
+    "    tol_fun=1e-10,\n",
+    "    tol_hmag_factor=1,\n",
+    "    precond_iter=10,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4879bf3b-f20c-451b-bcd7-84b7826a8b16",
+   "metadata": {},
+   "source": [
+    "## Putting all inputs in a Simulation object\n",
+    "\n",
+    "To define a `Simulation` object, we need to define a mesh, a `Materials` object, and a `Parameters` object as shown above."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "f87be0b7-49f9-4bc5-9075-561b04cc6e81",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sim = Simulation(\n",
+    "    mesh_filepath=CUBE_20_nm,\n",
+    "    materials=mat,\n",
+    "    parameters=par,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6053bd0b-449d-4366-9f60-8f8fff55a640",
+   "metadata": {},
+   "source": [
+    "Note that all of this could also have been defined using file paths:\n",
+    "```python\n",
+    "sim = Simulation(\n",
+    "    mesh_filepath=\"...\",\n",
+    "    materials_filepath=\"...\",\n",
+    "    parameters_filepath=\"...\",\n",
+    ")\n",
```

Comment:
Without further details this is very confusing. What am I supposed to put into the files? Maybe two short python scripts to define Material and Parameters?